lib/proxy/ssh/keys.c

```diff
@@ -3041,6 +3041,8 @@ static int decrypt_openssh_data(pool *p, const char *path,
   uint32_t buflen, key_len, rounds, salt_len, len = 0;
   size_t passphrase_len;
 
+  (void) len;
+
```

Comment:
These changes, not related to the ChaChaPoly implementation, are to quell compiler warnings about "set but not used" values.